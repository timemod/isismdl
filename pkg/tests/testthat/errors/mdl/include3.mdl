ident a = 1; 

#include incl/incl_err.mdl

a === 1 b = 2

#include incl/incl_err.mdl
