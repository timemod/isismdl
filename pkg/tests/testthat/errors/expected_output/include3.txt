Illegal #include syntax
Line: 3 of file mdl/include3.mdl
#include incl/incl_err.mdl
         ^

syntax error, unexpected '/'
Line: 3 of file mdl/include3.mdl
#include incl/incl_err.mdl
             ^

Illegal character in input
Line: 3 of file mdl/include3.mdl
#include incl/incl_err.mdl
                       ^

Illegal #include syntax
Line: 7 of file mdl/include3.mdl
#include incl/incl_err.mdl
         ^

Illegal character in input
Line: 7 of file mdl/include3.mdl
#include incl/incl_err.mdl
                       ^

